dts-shared-services/openshift/templates/backup-mongodb/backup-mongodb-deploy.param

```diff
@@ -0,0 +1,39 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/backup/backup-deploy.yaml
+#=========================================================
+NAME=backup-mongodb
+SOURCE_IMAGE_NAME=backup-mongodb
+NAMESPACE_NAME=a99fd4
+ROLE=backup-container
+SVC_PORT=27017
+APP_NAME=backup-container
+IMAGE_NAMESPACE=a99fd4-tools
+TAG_NAME=dev
+MONGODB_AUTHENTICATION_DATABASE=identity_kit_db
+BACKUP_STRATEGY=rolling
+WEBHOOK_URL=
+ENVIRONMENT_FRIENDLY_NAME=Digital Trust Demo Apps (dev)
+ENVIRONMENT_NAME=a99fd4-dev
+DATABASE_DEPLOYMENT_NAME=identity-kit-db-bc
```

Comment:
This parameter is missing form the other parameter files. Additionally, while I named it this way in the existing example, I think it would be better to make it "speaking" for the targeted DB since there could be more than one instance being backed up. So this could be something `IDENTITY_KIT_DB_SERVICE` or something like that.

Reply:
should I change the name in the mariadb configuration as well? or is the mariadb backup only applied to one database?

Reply:
Please update it if you can, that was an issue on my end - the pattern should be the same across templates. Maybe leave it as last thing OR even better in a separate PR so we don't have overlapping changes.